src/classes/PageControl.ts

```diff
@@ -74,11 +64,11 @@ export default class PageControl
      * Create a fieldset for the `passwordField`. This method will also look for an username field
      * @param passwordField The password field we're going to use
      */
-    private _createFieldSet(passwordField: HTMLElement): FieldSet | undefined
+    private _createFieldSet(passwordField: HTMLElement)
     {
-        let prevField: JQuery;
-        let fieldSet: FieldSet | undefined;
+        let prevField: JQuery<HTMLElement> | undefined;
         let $passwordField = $(passwordField);
+        let controlField: JQuery<HTMLElement> | undefined;
```

Comment:
```suggestion
        let prevField: JQuery | undefined;
        let $passwordField = $(passwordField);
        let controlField: JQuery | undefined;
```
Do you prefer the explicit type argument? I don't think it conveys any information here, since `JQuery` requires `HTMLElement` or a subclass.

Reply:
I prefer it like this right now, because I'm planning to drop JQuery in the future, so `JQuery<HTMLElement>` would become `HTMLElement`